worldwind/src/commonMain/kotlin/earth/worldwind/draw/DrawShapeState.kt

```diff
@@ -63,9 +125,8 @@ open class DrawShapeState internal constructor() {
 
     fun texCoordMatrix(matrix: Matrix3) = apply { texCoordMatrix.copy(matrix) }
 
-    fun texCoordAttrib(size: Int, offset: Int) = apply {
-        texCoordAttrib.size = size
-        texCoordAttrib.offset = offset
+    fun addAttribute (index: Int, vertexBuffer : AbstractBufferObject, size : Int, type : Int , normalized : Boolean, stride: Int, offset : Int) {
```

Comment:
Does it really is needed? Maybe make vertexcStatepublic and it will do?

Reply:
done